---
layout: page
title: Publications
---
## Project publications
To be posted...


## Relevant prior publications by PIs
Wang, M. & Maeda, H. A. Aromatic amino acid aminotransferases in plants. [Phytochem Rev](doi:10.1007/s11101-017-9520-6) 1–29 (2017). 

Wang, M., Toda, K. & Maeda, H. A. Biochemical properties and subcellular localization of tyrosine aminotransferases in Arabidopsis thaliana. Phytochemistry 132, 16–25 (2016).

Wang, M., Toda, K., Block, A. & Maeda, H. A. TAT1 and TAT2 tyrosine aminotransferases have both distinct and shared functions in tyrosine metabolism and degradation in Arabidopsis thaliana. J. Biol. Chem. 294, 3563–3576 (2019).

Dornfeld, C., Weisberg, A. J., K C, R., Dudareva, N., Jelesko, J. G. & Maeda, H. A. Phylobiochemical characterization of class-Ib aspartate/prephenate aminotransferases reveals evolution of the plant arogenate phenylalanine pathway. Plant Cell 26, 3101–3114 (2014).

Maeda, H., Yoo, H. & Dudareva, N. Prephenate aminotransferase directs plant phenylalanine biosynthesis via arogenate. Nat. Chem. Biol. 7, 19–21 (2011).

Holland, C. K., Berkovich, D. A., Kohn, M. L., Maeda, H. & Jez, J. M. Structural basis for substrate recognition and inhibition of prephenate aminotransferase from Arabidopsis. Plant J. 94, 304–314 (2018)

Northen, T. R., Yanes, O., Northen, M. T., Marrinucci, D., Uritboonthai, W., Apon, J., Golledge, S. L., Nordström, A. & Siuzdak, G. Clathrate nanostructures for mass spectrometry. Nature 449, 1033–1036 (2007).

Northen, T. R., Lee, J.-C., Hoang, L., Raymond, J., Hwang, D.-R., Yannone, S. M., Wong, C.-H. & Siuzdak, G. A nanostructure-initiator mass spectrometry-based enzyme activity assay. Proc. Natl. Acad. Sci. U.S.A. 105, 3678–3683 (2008).

Greving, M., Cheng, X., Reindl, W., Bowen, B., Deng, K., Louie, K., Nyman, M., Cohen, J., Singh, A., Simmons, B., Adams, P., Siuzdak, G. & Northen, T. Acoustic deposition with NIMS as a high-throughput enzyme activity assay. Anal Bioanal Chem 403, 707–711 (2012).

de Raad, M., de Rond, T., Rübel, O., Keasling, J. D., Northen, T. R. & Bowen, B. P. OpenMSI Arrayed Analysis Toolkit: Analyzing Spatially Defined Samples Using Mass Spectrometry Imaging. Anal. Chem. 89, 5818–5823 (2017).

Rübel, O., Greiner, A., Cholia, S., Louie, K., Bethel, E. W., Northen, T. R. & Bowen, B. P. OpenMSI: a high-performance web-based platform for mass spectrometry imaging. Anal. Chem. 85, 10354–10361 (2013).

Gao, J., Louie, K. B., Steinke, P., Bowen, B. P., Raad, M. de, Zuckermann, R. N., Siuzdak, G. & Northen, T. R. Morphology-Driven Control of Metabolite Selectivity Using Nanostructure-Initiator Mass Spectrometry. Anal. Chem. 89, 6521–6526 (2017).

Takasuka, T. E., Walker, J. A., Bergeman, L. F., Vander Meulen, K. A., Makino, S., Elsen, N. L. & Fox, B. G. Cell-free translation of biofuel enzymes. Methods Mol. Biol. 1118, 71–95 (2014).

Bianchetti, C. M., Takasuka, T. E., Deutsch, S., Udell, H. S., Yik, E. J., Bergeman, L. F. & Fox, B. G. Active site and laminarin binding in glycoside hydrolase family 55. J. Biol. Chem. 290, 11819–11832 (2015).

Deng, K., Takasuka, T. E., Heins, R., Cheng, X., Bergeman, L. F., Shi, J., Aschenbrener, R., Deutsch, S., Singh, S., Sale, K. L., Simmons, B. A., Adams, P. D., Singh, A. K., Fox, B. G. & Northen, T. R. Rapid kinetic characterization of glycosyl hydrolases based on oxime derivatization and nanostructure-initiator mass spectrometry (NIMS). ACS Chem. Biol. 9, 1470–1479 (2014).

Yoshikuni, Y. & Keasling, J. D. Pathway engineering by designed divergent evolution. Curr Opin Chem Biol 11, 233–239 (2007).

Yoshikuni, Y., Martin, V. J. J., Ferrin, T. E. & Keasling, J. D. Engineering cotton (+)-delta-cadinene synthase to an altered function: germacrene D-4-ol synthase. Chem. Biol. 13, 91–98 (2006).

Yoshikuni, Y., Ferrin, T. E. & Keasling, J. D. Designed divergent evolution of enzyme function. Nature 440, 1078–1082 (2006).

Arnold, A. & Nikoloski, Z. Bottom-up Metabolic Reconstruction of Arabidopsis and Its Application to Determining the Metabolic Costs of Enzyme Production. Plant Physiol. 165, 1380–1391 (2014).

Arnold, A., Sajitz-Hermstein, M. & Nikoloski, Z. Effects of varying nitrogen sources on amino acid synthesis costs in Arabidopsis thaliana under different light and carbon-source conditions. PLoS ONE 10, e0116536 (2015).

Szecowka, M., Heise, R., Tohge, T., Nunes-Nesi, A., Vosloh, D., Huege, J., Feil, R., Lunn, J., Nikoloski, Z., Stitt, M., Fernie, A. R. & Arrivault, S. Metabolic fluxes in an illuminated Arabidopsis rosette. Plant Cell 25, 694–714 (2013).

Eloundou-Mbebi, J. M. O., Küken, A., Omranian, N., Kleessen, S., Neigenfind, J., Basler, G. & Nikoloski, Z. A network property necessary for concentration robustness. Nat Commun 7, (2016).

Heise, R., Arrivault, S., Szecowka, M., Tohge, T., Nunes-Nesi, A., Stitt, M., Nikoloski, Z. & Fernie, A. R. Flux profiling of photosynthetic carbon metabolism in intact plants. Nat Protoc 9, 1803–1824 (2014).

Küken, A. & Nikoloski, Z. Computational Approaches to Design and Test Plant Synthetic Metabolic Pathways. Plant Physiol. 179, 894–906 (2019)

Töpfer, N., Caldana, C., Grimbs, S., Willmitzer, L., Fernie, A. R. & Nikoloski, Z. Integration of genome-scale modeling and transcript profiling reveals metabolic pathways underlying light and temperature acclimation in Arabidopsis. Plant Cell 25, 1197–1211 (2013).




<img src="../img/pubs/journal-article.png" height="20px"> **First Author**, author 2, author 3. *Title* [journal](https://doi.org/)

_______________

<sup>co</sup> designates co-first author
